Description: add search paths for kdelibs4 docs
Author: Jonathan Riddell
Origin: me
Forwarded: no, kdelibs4 path was set in debian/kubuntu packages
Applied-Upstream: nope
Reviewed-by: Jonathan Riddell
Last-Update: 2015-03-01
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- kio/src/ioslaves/help/kio_help.cpp	2015-03-07 15:45:21.000000000 +0100
+++ kio/src/ioslaves/help/kio_help.cpp	2015-04-02 17:15:39.687828131 +0200
@@ -49,7 +49,8 @@
     QStringList search;
 
     // assemble the local search paths
-    const QStringList localDoc = QStandardPaths::locateAll(QStandardPaths::GenericDataLocation, "doc/HTML", QStandardPaths::LocateDirectory);
+    QStringList localDoc = QStandardPaths::locateAll(QStandardPaths::GenericDataLocation, "doc/HTML", QStandardPaths::LocateDirectory);
+    localDoc << QStandardPaths::locateAll(QStandardPaths::GenericDataLocation, "doc/kde/HTML", QStandardPaths::LocateDirectory);
 
     QStringList langs = QLocale().uiLanguages();
     langs.append("en");
